app/uk/gov/hmrc/apigatekeeperapprovalsfrontend/controllers/TermsOfUseHistoryController.scala

```diff
@@ -93,6 +102,14 @@ class TermsOfUseHistoryController @Inject() (
       }
     }
 
+    def deriveInvitationStatusDescription(status: TermsOfUseInvitationState): String = {
+      status match {
+        case REMINDER_EMAIL_SENT => "We sent a reminder email to the admins of the application."
+        case OVERDUE             => "The terms of use has not been completed by the due date."
```

Comment:
Grammar looks wrong. "terms" is plural, so should be "have" not "has" ?
Alternatively, "The terms of use acceptance has not been ...."

Reply:
Fixed